static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -109,6 +109,10 @@
     .adverts__row > * + *::before {
         content: none;
     }
+
+    .adverts__badge__help {
```

Comment:
It's ugly. Isn't there any other way than creating an element of element? There is a new _brandbadge.scss object. Maybe it should be something like that. `.brandbadge__help` then makes sense.